flake.nix

```diff
@@ -0,0 +1,35 @@
+{
+  inputs.flake-utils.url = github:numtide/flake-utils;
+  outputs = { self, nixpkgs, flake-utils }:
+  flake-utils.lib.eachSystem flake-utils.lib.defaultSystems (system:
```

Comment:
note that I had to change to defaultSystems,
allSystems does not work for you, you are not yet building to all possible systems.